package/MDAnalysis/coordinates/base.py

```diff
@@ -1072,7 +1073,7 @@ class ProtoReader(six.with_metaclass(_Readermeta, IObase)):
     def __init__(self):
         # initialise list to store added auxiliary readers in
         # subclasses should now call super
-        self._auxs = {}
+        self._auxs = OrderedDict()
```

Comment:
Modifications might be order sensitive, so use OrderedDict to keep track of who was added first.